### ISSUE TITLE
how to calculate MSE loss?

### ISSUE THREAD

**Issue by cnaaq** [natanielruiz/deep-head-pose#46]
Hi,

In calculating MSE loss, a series of code is written. My question is about the third line:
`yaw, pitch, roll = model(image)`
...
`yaw_predicted = softmax(yaw)`
...
`yaw_predicted = torch.sum(yaw_predicted * idx_tensor, 1) * 3 - 99  # where did this formula come from? what is the logic behind it?`
...
`loss_reg_yaw = reg_criterion(yaw_predicted, label_yaw_cont)`

Thanks in advance

**Comment by natanielruiz** [comment]
To understand this please check the paper!
